File: qqbot-app/src/main/java/com/mieai/qqbot/app/QqBotApplication.java
package com.mieai.qqbot.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.SpringBootConfiguration;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;

@SpringBootConfiguration
@EnableAutoConfiguration
public class QqBotApplication {

    public static void main(String[] args) {
        SpringApplication.run(QqBotApplication.class, args);
    }
}

File: qqbot-app/src/main/kotlin/com/mieai/qqbot/app/QqBotApplication.kt
package com.mieai.qqbot.app

import org.springframework.boot.SpringApplication
import org.springframework.boot.SpringBootConfiguration
import org.springframework.boot.autoconfigure.EnableAutoConfiguration

@SpringBootConfiguration
@EnableAutoConfiguration
class QqBotApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(QqBotApplication::class.java, *args)
        }
    }
}
